api/lib/infrastructure/repositories/schooling-registration-repository.js

```diff
@@ -39,6 +40,27 @@ function _setSchoolingRegistrationFilters(qb, { lastName, firstName, connexionTy
   }
 }
 
+function _isReconciliate(schoolingRegistration) {
+  return schoolingRegistration.userId;
+}
+
+function _update(schoolingRegistrationsToUpdate, trx, organizationId) {
+  console.time('Update');
+  return bluebird.mapSeries(schoolingRegistrationsToUpdate, async (schoolingRegistrationToUpdate) => {
+    await trx('schooling-registrations')
+      .where({
+        'organizationId': organizationId,
+        'nationalStudentId': schoolingRegistrationToUpdate.nationalStudentId,
+      })
+      .update(_.omit(schoolingRegistrationToUpdate, ['id', 'createdAt']));
+  }).then(function() { console.timeEnd('Update'); });
+}
+
+function _create(schoolingRegistrationsToCreate) {
+  console.time('Create');
+  return Bookshelf.knex.batchInsert('schooling-registrations', schoolingRegistrationsToCreate).then(function() { console.timeEnd('Create');});
```

Comment:
Attention, le `return` retourne `undefined`.
Est-ce volontaire ?
```suggestion
  await Bookshelf.knex.batchInsert('schooling-registrations', schoolingRegistrationsToCreate).
  console.timeEnd('Create');
```